Composer/packages/client/src/pages/botProject/BotProjectSettingsTableView.tsx

```diff
@@ -40,12 +42,15 @@ export const BotProjectSettingsTableView: React.FC<RouteComponentProps<{
   const botProjects = useRecoilValue(localBotsDataSelector);
   const botProject = botProjects.find((b) => b.projectId === projectId);
   const isRootBot = !!botProject?.isRootBot;
+  const useAdapters = useFeatureFlag('NEW_CREATION_FLOW');
```

Comment:
There is a constant for NEW_CREATION_FLOW in featureFlags.ts. The cool thing about that is that when this feature flag changes name or is deleted, this line would throw an error and we'd detect that we need to update the code.
Not sure if feature flags is accessible from here but worth considering.

Reply:
Good catch. I'll fix that.

Reply:
On second look, we don't need to. `useFeatureFlag` already takes a value of type `FeatureFlagKey` as an argument, so if that definition changes, this line will no longer typecheck, and that's exactly the error you mentioned.